packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -410,7 +410,7 @@ void main() {
     testUsingContext('prints warning if macOS Podfile parses .flutter-plugins', () async {
       final FlutterProject projectUnderTest = setupProjectUnderTest();
       pretendPodIsInstalled();
-      pretendPodVersionIs('1.10.0');
+      pretendPodVersionIs('100.0.0');
```

Comment:
These tests require a sufficient version is installed.  Up the version to a really high number so we don't need to change these tests every time we bump the recommended or required versions.